src/UI/Factory.php

```diff
@@ -292,4 +292,17 @@ public function legacy($content);
 	 * @return \ILIAS\UI\Component\Panel\Factory
 	 */
 	public function panel();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     View Controls switch between different visualisation of data.
+	 *   composition: >
+	 *      View Controls are composed mainly of buttons, they are often found in toolbars.
+	 *   effect: Interacting with a view control changes to display in some content area.
```

Comment:
Shouldn't there be some rule about context switches here? I remember some discussion we had about view control vs. navigation and how to separate those two components.